src/fsharp/PostInferenceChecks.fs

```diff
@@ -455,6 +455,30 @@ let CheckMultipleInterfaceInstantiations cenv interfaces m =
     | Some (typ1,typ2) -> 
          errorR(Error(FSComp.SR.chkMultipleGenericInterfaceInstantiations((NicePrint.minimalStringOfType cenv.denv typ1), (NicePrint.minimalStringOfType cenv.denv typ2)),m))
 
+// tries to extract the name of an expression
+let extractNameOf args =
```

Comment:
If it tries to extract the name of an expression, can we name it clearly with `tryExtractNameOf`